.github/workflows/CICD.yml

```diff
@@ -0,0 +1,347 @@
+name: CICD
+
+# spell-checker:ignore (acronyms) CICD MSVC musl
+# spell-checker:ignore (names) CodeCOV MacOS MinGW Peltoche rivy sharkdp
+# spell-checker:ignore (jargon) SHAs softprops toolchain
+# spell-checker:ignore (shell/tools) choco clippy dmake dpkg esac fakeroot gmake halium libssl mkdir nullglob popd printf pushd rustc rustfmt rustup shopt
+# spell-checker:ignore (misc) gnueabihf issuecomment maint onexitbegin onexitend uutils
```

Comment:
This seems to be some editor/tool specific section. I would prefer if we could remove it to keep this file simple.

Reply:
This was a quick conversion from another repository.
I'll try to trim it down for you and update the PR tomorrow.

Reply:
I'm testing an updated commit right now. I push it in a few minutes barring any issues.